src/main/java/uk/gov/companieshouse/mapper/email/DissolutionEmailMapper.java

```diff
@@ -61,6 +63,24 @@ public ApplicationRejectedEmailData mapToApplicationRejectedEmailData(
         applicationRejectedEmailData.setCompanyNumber(dissolution.getCompany().getNumber());
         applicationRejectedEmailData.setCompanyName(dissolution.getCompany().getName());
         applicationRejectedEmailData.setRejectReasons(rejectReasons);
+        applicationRejectedEmailData.setPaymentReference(dissolution.getPaymentInformation().getReference());
+
+        return applicationRejectedEmailData;
+    }
+
+    public ApplicationRejectedEmailData mapToApplicationRejectedEmailDataCopy(
```

Comment:
I wouldn't duplicate this mapper method. I'd reuse the original reject email mapper method, and make it take an `emailAddress` parameter, so it's reusable

Reply:
```
public ApplicationRejectedEmailData mapToApplicationRejectedEmailData(
            Dissolution dissolution, List<String> rejectReasons, Optional<String> forceEmail
    ) 
```
How is this?

Reply:
Rather than `forceEmail`, I'd just simply call it `emailAddress`.
And assuming it's optional because if an email address isn't passed to the method, it'll default to the authenticated user's email address?

Reply:
if you pass it as null it uses the dissolution users email